### PR TITLE
Show ACMG classification for pinned variants

### PR DESCRIPTION
**Pull request by mhkc**
Show the ACMG classification for pinned variants

image

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
RD case:

Master:

image

This branch:

image


Adding another classified pinned variant: 
image

File: scout/server/blueprints/cases/templates/cases/utils.html [hunk 1]
```diff
@@ -520,6 +520,7 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
                     {% if case.track == 'cancer' %}
                       {{ tier_cell(variant, manual_rank_options, cancer_tier_options) }}
                     {% else %}
+                      {{ acmg_classification_badge }}
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/utils.html:523]:
```suggestion
                      {{ acmg_classification_badge(variant) }}
```

**Review by northwestwitch — CHANGES_REQUESTED** [review]
You forgot to pass the variant

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Warning on overwriting variants with same position was no longer shown
 - Increase the height of the dropdowns to 425px
 - More indices for the case table as it grows, specifically for causatives queries
+- Show ACMG classification for pinned variants in case view
```
Comment (northwestwitch) [inline CHANGELOG.md:15]:
```suggestion
- Show manual ACMG classification for pinned variants in case view
```

**Review by northwestwitch — CHANGES_REQUESTED** [review]

File: scout/server/blueprints/variants/templates/variants/indicators.html [hunk 3]
```diff
@@ -73,6 +73,16 @@
 {% endmacro %}
 
 
+{% macro acmg_classification_badge(variant) %}
```
[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/server/blueprints/variants/templates/variants/indicators.html:76]: Yes: if you have the code in a separate macro, case page looks like this:

image

But variantS page doesn't show the badge:

image


But if you include the code of this macro in the macro above:

image

File: scout/server/blueprints/cases/templates/cases/utils.html [hunk 4]
```diff
@@ -520,6 +520,7 @@ <h5 class="modal-title" id="exampleModalLabel">Beacon submission</h5>
                     {% if case.track == 'cancer' %}
                       {{ tier_cell(variant, manual_rank_options, cancer_tier_options) }}
                     {% else %}
+                      {{ acmg_classification_badge(variant) }}
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/utils.html:523]: 👍🏻

**Review by northwestwitch — APPROVED** [review]

**Comment by dnil** [comment]
Sorry to barge in, but I'm still seeing double ACMG calls for the pinned ones? That cannot be intended, right? We usually stick with showing the classification, regardless if it was made using the tool or just manually entered - though there are some old issue there somewhere.

**Comment by northwestwitch** [comment]
> Sorry to barge in, but I'm still seeing double ACMG calls for the pinned ones? That cannot be intended, right? We usually stick with showing the classification, regardless if it was made using the tool or just manually entered - though there are some old issue there somewhere.

Yeah I don't know. Perhaps he was asked to display in a badge how the classification was done?

**Comment by mhkc** [comment]
I had some reports that ACMG classifications made with scout were missing in the pinned variants. They seem not to be included in the `clinical_assessments` and they are just present in `acmg_classification`

**Comment by northwestwitch** [comment]
OK let's perhaps wait to merge this one. Let's make sure that the functionality doesn't work already and what and where can be improved

**Comment by northwestwitch** [comment]
@mhkc to answer your question the other badge is shown when you click on the acmg classsification buttons or you use the tool by clicking on the button immediately under. So the code of this PR looks redundant.

image

BUT!

I've checked what could be wrong with the missing ACMG for pinned variants and I've found out that pinned classified STRs don't actually have an ACMg badge on case page. Look for instance at this example: top variant is a SNV, bottom variant is a STR. Both with ACMG classification:

image

Could this be the original problem found by clinicians in Lund?

**Comment by dnil** [comment]
Formally STRs don't have ACMG. They could and should have other variant tags. I guess pragmatically, someone may be using the ACMG classification for them anyway, treating it more as guidelines. https://gph.is/g/EJmDXnV

**Comment by northwestwitch** [comment]
> Formally STRs don't have ACMG. They could and should have other variant tags. I guess pragmatically, someone may be using the ACMG classification for them anyway, treating it more as guidelines.
> image

I knew this comment was coming from the moment I wrote mine! 😆

**Comment by northwestwitch** [comment]
I've realized that we actually have a bug (see also issue #2710) causing not only ACMG classification to be missing in pinned panel, but also dismissed status, manual rank, tier and mosaic status missing in causatives panel as well. 
I've written a PR (#2713) that should cover all this, so I'm closing this one.

**Pull request closed without merging**